entity.go

```diff
@@ -55,6 +53,15 @@ func (ck ClusteringKey) String() string {
 	return ck.Name + " ASC"
 }
 
+// Format a key's ClusteringKeys.
+func formatClusteringKeys(keys []*ClusteringKey) string {
+	pieces := make([]string, len(keys))
+	for index, ck := range keys {
```

Comment:
Does order matter? Seems like this method will return different results for the same keyset.